Jenkinsfile

```diff
@@ -0,0 +1,3 @@
+Map pipelineParams = [ "mavenAdditionalArgs" : "-U -Djava.net.preferIPv4Stack=true -Dsurefire.rerunFailingTestsCount=5 -Dmaven.test.failure.ignore=true" ]
```

Comment:
It probably works without any of these additional arguments

Reply:
maven.test.failure.ignore=true this one sure that is not needed as it is added by default in the mvn() step

Reply:
Run the tests without any of these additional arguments and it was successful